pyproject.toml

```diff
@@ -30,6 +30,7 @@ python-json-logger = "^2.0.2"
 django-storages = { version = "^1.11.1", extras = ["google"] }
 "pdfminer.six" = "^20211012"
 requests = "2.27.1"
+django-sass = "1.1.0"
```

Comment:
Adding quotes to the library name \*might\* fix the build error, e.g.:

`"django-sass" = "1.1.0"`

Reply:
Implementeed this.